hive/src/main/java/org/apache/iceberg/hive/HiveCatalog.java

```diff
@@ -188,6 +305,33 @@ protected String defaultWarehouseLocation(TableIdentifier tableIdentifier) {
         tableIdentifier.name());
   }
 
+  public  Map<String, String> getMetafrpmhiveDb(Database database) {
+    Map<String, String> nameMap = new HashMap<>();
+
+    nameMap.put("name", database.getName());
+    nameMap.put("location", database.getLocationUri());
+    nameMap.put("comment", database.getDescription());
+    database.getParameters().forEach(nameMap::put);
+
+    return nameMap;
```

Comment:
Also what is the metadata here?

Reply:
In  SQL syntax rules: `CREATE NAMESPACE namespace_name [COMMENT 'namespace_comment']` , the `COMMENT` stored into the hiveMetaStore, I think while read the `Description`  maybe should transform it to  `comment`  for user .